[BUG] NullPointerException when reset-stuck-builds-actor triggers

Hi,
as per title, there is a `NullPointerException` when the actor that resets the builds that are stuck triggers.
This is the stacktrace:
```java
[ERROR] [10/06/2020 16:58:15.311] [swarm-plugin-akka.actor.default-dispatcher-61] [akka://swarm-plugin/user/reset-stuck-builds-actor] null
java.lang.NullPointerException
        at org.jenkinsci.plugins.docker.swarm.ResetStuckBuildsInQueueActor.resetStuckBuildsInQueue(ResetStuckBuildsInQueueActor.java:32)
        at org.jenkinsci.plugins.docker.swarm.ResetStuckBuildsInQueueActor.lambda$createReceive$0(ResetStuckBuildsInQueueActor.java:23)
        at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:24)
        at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:20)
        at scala.PartialFunction.applyOrElse(PartialFunction.scala:127)
        at scala.PartialFunction.applyOrElse$(PartialFunction.scala:126)
        at akka.japi.pf.UnitCaseStatement.applyOrElse(CaseStatements.scala:20)
        at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:175)
        at akka.actor.Actor.aroundReceive(Actor.scala:539)
        at akka.actor.Actor.aroundReceive$(Actor.scala:537)
        at akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:227)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:612)
        at akka.actor.ActorCell.invoke(ActorCell.scala:581)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:268)
        at akka.dispatch.Mailbox.run(Mailbox.scala:229)
        at akka.dispatch.Mailbox.exec(Mailbox.scala:241)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

I think that the bug is due to a `null` being returned when `DockerSwarmCloud.get()` is invoked.

--- a/src/main/java/org/jenkinsci/plugins/docker/swarm/ResetStuckBuildsInQueueActor.java
+++ b/src/main/java/org/jenkinsci/plugins/docker/swarm/ResetStuckBuildsInQueueActor.java
@@ -2,6 +2,7 @@ package org.jenkinsci.plugins.docker.swarm;
 
 import java.io.IOException;
 import java.util.Date;
+import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 import java.util.logging.Logger;
 
@@ -16,6 +17,7 @@ import scala.concurrent.duration.Duration;
 
 public class ResetStuckBuildsInQueueActor extends AbstractActor {
     private static final Logger LOGGER = Logger.getLogger(ResetStuckBuildsInQueueActor.class.getName());
+    private static final long DEFAULT_RESET_MINUTES = 60L;
     public static final int CHECK_INTERVAL = 1;
 
     @Override
@@ -24,12 +26,12 @@ public class ResetStuckBuildsInQueueActor extends AbstractActor {
     }
 
     public static Props props() {
-        return Props.create(ResetStuckBuildsInQueueActor.class, () -> new ResetStuckBuildsInQueueActor());
+        return Props.create(ResetStuckBuildsInQueueActor.class, ResetStuckBuildsInQueueActor::new);
     }
 
     private void resetStuckBuildsInQueue() throws IOException {
         try {
-            final long RESET_MINUTES = DockerSwarmCloud.get().getTimeoutMinutes();
+            long resetMinutes = Optional.ofNullable(DockerSwarmCloud.get().getTimeoutMinutes()).orElse(DEFAULT_RESET_MINUTES);
             final Queue.Item[] items = Jenkins.getInstance().getQueue().getItems();
             for (int i = items.length - 1; i >= 0; i--) { // reverse order
                 final Queue.Item item = items[i];
@@ -40,7 +42,7 @@ public class ResetStuckBuildsInQueueActor extends AbstractActor {
                 if (lblAssignmentAction != null) {
                     long inQueueForMinutes = TimeUnit.MILLISECONDS
                             .toMinutes(new Date().getTime() - lblAssignmentAction.getProvisionedTime());
-                    if (inQueueForMinutes > RESET_MINUTES) {
+                    if (inQueueForMinutes > resetMinutes) {
                         final String computerName = lblAssignmentAction.getLabel().getName();
                         final Node provisionedNode = Jenkins.getInstance().getNode(computerName);
                         if (provisionedNode != null) {
@@ -52,12 +54,12 @@ public class ResetStuckBuildsInQueueActor extends AbstractActor {
                 }
             }
         } finally {
-            resechedule();
+            reschedule();
         }
 
     }
 
-    private void resechedule() {
+    private void reschedule() {
         ActorSystem system = getContext().getSystem();
         system.scheduler().scheduleOnce(Duration.apply(CHECK_INTERVAL, TimeUnit.MINUTES), getSelf(), "restart",
                 getContext().dispatcher(), ActorRef.noSender());

